components/usage-api/usage/v1/billing.proto

```diff
@@ -50,3 +53,14 @@ message FinalizeInvoiceRequest {
 
 message FinalizeInvoiceResponse {
 }
+
+// If there are two billable sessions for this instance ID,
+// the second one's "from" will be the first one's "to"
+message SetBilledSessionRequest {
+  string instance_id = 1;
+  google.protobuf.Timestamp from = 2;
+  string system = 3;
```

Comment:
I think the billing system here should be a [oneof](https://developers.google.com/protocol-buffers/docs/proto3#oneof) to better encode the options there are.